src/app/level/level-list/level-list.component.html

```diff
@@ -0,0 +1,9 @@
+<body>
+<h1>Levels</h1>
+<ul *ngIf="levels" class="my-div">
+  <li *ngFor ="let level of levels; let i = index" [class.selected]="level.levelId === selectedId" (click)="onSelect(level)">
+    <a [routerLink]="['/levels', i+1] " class="fill-div"> Level {{i+1}}: {{ level.name }}</a>
```

Comment:
This is prone to failure, the corresponding level id will not always be `i+1`. You should use the level `id` there. I know `level.levelId` is `undefined`, it's because the property is not called `levelId` but just `id` and the Typescript class attributes must match the JSON ones.